cluster/manager_delete.go

```diff
@@ -247,6 +248,17 @@ func deleteDnsRecordsOwnedByCluster(cluster CommonCluster) error {
 	return nil
 }
 
+func deleteUnusedSecrets(cluster CommonCluster) error {
```

Comment:
Make this a method of `Manager` and use the logger that `Manager` holds.

Reply:
How about the solution I added in [this](https://github.com/banzaicloud/pipeline/pull/1638/commits/c7c2fb5c11448722be0adb857cee563aeea69603) commit? This way, this function uses the logger of `Manager` (as it gets it as a parameter), but I also kept the way how similar functions are implemented in this file, like e.g. [DeleteAllResources()](https://github.com/banzaicloud/pipeline/blob/c7c2fb5c11448722be0adb857cee563aeea69603/cluster/manager_delete.go#L72) or[ deleteDnsRecordsOwnedByCluster()](https://github.com/banzaicloud/pipeline/blob/c7c2fb5c11448722be0adb857cee563aeea69603/cluster/manager_delete.go#L233), so to keep it as a simple function in this file and not a method of `Manager`. Do you think this is reasonable like this?